console/console-init/ui/src/modules/iot-certificates/components/CertificateCard/CertificateCard.tsx

```diff
@@ -41,14 +42,22 @@ const styles = StyleSheet.create({
 export const CertificateCard: React.FunctionComponent<ICertificateCardProps> = ({
   certificate,
   setOnEditMode,
+  onDelete,
+  onEnableOrDisable,
   id
 }) => {
   const handleEditCertificate = () => {
     setOnEditMode(true);
   };
 
   const handleDeleteCertificate = () => {
-    // TODO: Mutation to delete the certificate
+    onDelete(certificate);
+  };
+  const onEnableChange = (
```

Comment:
As you are following handle prfix with other functions. I would like to suggest function name handleCertificateStaus instead of  onEnableChange.